rs/common/comparators/shield_and_sword_comparator.py

```diff
@@ -0,0 +1,14 @@
+from rs.common.comparators.common_general_comparator import default_comparisons, add_to_comparison_list, \
+    move_in_comparison_list, CommonGeneralComparator
+from rs.common.comparators.core.comparisons import *
+
+comparisons = default_comparisons.copy()
+comparisons.remove(lowest_health_monster)
+add_to_comparison_list(comparisons, comparison_to_add=spear_lowest_health, after=most_dead_monsters)
+
+
+# Difference to normal comparator:
+# Go very aggressive on killing either the front or back sentry for as long as there are 3 sentries alive.
```

Comment:
nit: Comment is copy paste and not relevant

Reply:
Oops